psalm.xml

```diff
@@ -13,6 +13,9 @@
 			<directory name="widgets" />
 		</ignoreFiles>
 	</projectFiles>
+	<stubs>
+		<file name="sensei-lms.php" />
```

Comment:
We had lots of errors like `Undefined function Sensei()`. I encountered it in this PR as well.
It looks like Psalm for some reason can't discover some entities via reflection. In these cases, I found a recommendation, put files containing those entities into `<stubs>`.
Another approach might be to create `config/psalm/psalm-stubs.php` and define functions like `Sensei()` there (without implementation) and include these stubs in config like I did with above.